tests/components/tplink/test_light.py

```diff
@@ -301,8 +301,8 @@ async def test_smartswitch(
 
     state = hass.states.get("light.dimmer1")
     assert state.state == "on"
-    assert state.attributes["brightness"] == 53.55
-    assert sys_info["brightness"] == 21
+    assert state.attributes["brightness"] == 56
+    assert sys_info["brightness"] == 22
```

Comment:
Why the second change?

Reply:
Since it was using int() it would drop all the floating point precision.  Depending on how the math worked out it could be off by 0,1, or 2 on the round trip.

The second change is because its now rounding up since its closer to 22 than 21.